docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,162 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### High Availability in PMM
+
+
+We are excited to share our roadmap for High Availability (HA) in PMM, a new solution that our users have been anticipating. HA provides a unique architectural design with near-zero downtime for your PMM setup and ensures monitoring without data loss during a failover.
+
+To ensure a smooth rollout, PMM HA will be introduced in phases, in line with your requirements and feedback, with PMM 2.33.0 being the starting point. 
+
+!!! caution alert alert-warning "Important"
+    PMM HA will be rolled out in three stages. Stage one will be included with PMM 2.33.0 and stages two and three will be rolled out next year.
+
+The following features/use cases will be covered as part of the various stages of HA in PMM:
+
+**Stage 1**
+
+As a first step toward preventing data loss, we have developed the following solution:
+
+**Prevent data loss with VM integration for short outages**
```

Comment:
this should be somehow better indented - as now it is the same indent level as `Stage ` . Maybe shifted a little bit right. same for other stages. As it hard to follow what is what.